cl/users/models.py

```diff
@@ -190,8 +259,211 @@ def generate_recap_email(user_profile: UserProfile, append: int = None) -> str:
     return recap_email
 
 
+def handle_hard_bounce(
+    message_id: str, bounceSubType: str, recipient_emails: List[str]
+) -> None:
+    """Handle a hard bounce notification received from SNS
+    :param message_id: The unique message id assigned by Amazon SES
+    :param bounceSubType: The subtype of the bounce, as determined
```

Comment:
I think I'd convert this to snake_case instead of camelCase even though AWS prefers to camelCase.

Reply:
Done!